### DISCUSSION TITLE
How to use with Sublime Text

### DISCUSSION THREAD

**Discussion by href** [ansible/ansible-language-server#503] [Q&A]
I'm unable to get this to work with Sublime Text. I'm trying to set this up using the custom client configuration mechanism:
https://lsp.sublimetext.io/client_configuration/

I installed the language server using `npm install -g @ansible/ansible-language-server`, and verified that starting it using `ansible-langauge-server --stdio` works (there's no output though).

I then tried to run it in various ways:

1. With a path environment set:

```json
{
    "log_debug": true,
    "clients":
    {
        "ansible": {
          "enabled": true,
          "command": ["/opt/homebrew/bin/ansible-language-server", "--stdio"],
          "selector": "source.ansible",
          "env": {
            "PATH": "/Users/denis/.projects/sys/venv/bin/"
          }
        }
    }
}
```

2. With settings:

```json
{
    "log_debug": true,
    "clients":
    {
        "ansible": {
          "enabled": true,
          "command": ["/opt/homebrew/bin/ansible-language-server", "--stdio"],
          "selector": "source.ansible",
          "env": {
            "PATH": "/Users/denis/.projects/sys/venv/bin/"
          },
          "settings": {
            "ansible.python.interpreterPath": "/Users/denis/.projects/sys/venv/bin/python3",
            "ansible.ansible.path": "/Users/denis/.projects/sys/venv/bin/ansible",
          }
        }
    }
}
```

In both cases, I can see that the ansible-langauge-server is briefly started when I open an Ansible YAML file, but the process vanishes immediately and from what I can tell there is no LSP functionality.

In other projects I use LSP-rust-analyzer for Sublime Text, which works fine.

Does anyone have a working Sublime Text config they would be willing to share?

**Comment by p3lim** [comment]
@href Did you manage to get this working?
